repository/volunteer_unavailability_v2.py

```diff
@@ -23,3 +28,35 @@ def edit_event(session, userId, eventId, title=None, start=None, end=None, perio
         session.rollback()
         logging.error(e)
         return None
+
+def get_event(session, userId):
```

Comment:
Can you wrap this in a class called "EventRepository" and inject it through the constructor of the unavailability api resource? For unit testing reasons